api/controller/controller.go

```diff
@@ -189,6 +189,27 @@ func (c *Client) WatchAllModels() (*api.AllWatcher, error) {
 	return api.NewAllModelWatcher(c.facade.RawAPICaller(), &info.AllWatcherId), nil
 }
 
+// WatchModelSummaries returns a SummaryWatcher, from which you can request
+// the Next set of ModelAbstracts for all models the user can see.
```

Comment:
As an aside, why are there two methods for this? From a client point of view, if you are a superuser then the set of models you can see is all the models on the controller. Therefore you would get the same results for either method?

Reply:
I specifically split this in two because even though I may be a superuser in a controller, perhaps I only want to see "my" models. Without this there would be no way to support this mechanism.

It also simplifies the authentication and internals to be explicit.